core/src/test/java/org/apache/cxf/helpers/ServiceUtilsTest.java

```diff
@@ -173,4 +171,23 @@ private void setupSchemaValidationValue(Object value, boolean isRequestor) {
 
         control.replay();
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testMakeNamespaceFromClassName() {
+        String nameSpace = ServiceUtils.makeNamespaceFromClassName("Foo", "http");
+        assertEquals("http://DefaultNamespace", nameSpace);
+
+        nameSpace = ServiceUtils.makeNamespaceFromClassName(this.getClass().getName(), "http");
+        assertEquals("http://helpers.cxf.apache.org/", nameSpace);
+
+    }
+
+    @Test
+    public void testMakePackageName() {
+        String packageName = ServiceUtils.makePackageName("http://www.w3.org/TR/html4/");
+        assertEquals("org.w3.www.TR.html4", packageName);
+
+        packageName = ServiceUtils.makePackageName("http://www.w3.org");
```

Comment:
Also better to have 2 test cases for different asserts

Reply:
@reta . Done

Reply:
Thank you